changelogs/unreleased.md

```diff
@@ -0,0 +1,40 @@
+# Changelog Template
+
+The format is based on [Rust's Release Notes](https://github.com/rust-lang/rust/blob/master/RELEASES.md),
+with the exception of splitting the changelog into multiple files for readability purposes.
+
+## [Unreleased]
+
+### Language
+
+- Aliases
+- Arrays of Unspecified Size
+- Explicit `const function` definitions
+- Countdown loops
```

Comment:
I think we can be a little more elaborate in each item... e.g., say that the syntax is `for i in 0..10` and so on.

Reply:
I personally don't think such things belong in the release note. Rather they should link to documentation about the changes, but we don't have a link till we update and release the documentation pages for the changes.